src/Controller/EncryptionProfileListBuilder.php

```diff
@@ -31,6 +72,20 @@ public function buildRow(EntityInterface $entity) {
     $row['label'] = $this->getLabel($entity);
     $row['id'] = $entity->id();
     $row['key'] = $entity->getEncryptionKey();
+
+    if ($this->config->get('check_profile_status')) {
+      $errors = $entity->validate();
```

Comment:
I'm not sure how this works. Wouldn't it need to validate against some testing string or something? I envision this mimics someone trying to perform encrypt/decrypt with an encryption profile